package.json

```diff
@@ -15,7 +15,7 @@
     "rmdir": "node ./scripts/rmdir.mjs",
     "ts-declaration": "tsc --emitDeclarationOnly --outDir dist/types",
     "did": "scripts/compile-idl-js",
-    "build": "npm run rmdir && node esbuild.mjs && npm run ts-declaration",
+    "build": "npm run rmdir && mkdir -p dist && cp -R proto candid dist && node esbuild.mjs && npm run ts-declaration",
```

Comment:
Note: without this esm builds fail.  Cjs builds SHOULD probably fail as well, because of the missing code, but they don't for some reason.  Lazy loading? Tree shaking?